ArjunaJTS/jts/classes/com/arjuna/ats/internal/jts/orbspecific/interposition/resources/arjuna/ServerTopLevelAction.java

```diff
@@ -591,6 +591,9 @@ protected boolean registerResource (Coordinator theCoordinator)
 
 	if (theCoordinator != null)
 	{
+	    if (resourceRefRegistered)
```

Comment:
I think we saw it going via SSTLA in the debugger (it was the second call to register_resource). As such I wondered if now you were adding protection to the base class it is no longer required.

If your analysis has shown that this is still required and the base class protection is not required then that is good for me.